src/components/Image/index.js

```diff
@@ -7,31 +7,23 @@ import {defaultProps, imagePropTypes} from './imagePropTypes';
 import RESIZE_MODES from './resizeModes';
 
 class Image extends React.Component {
-    constructor(props) {
-        super(props);
-
-        this.state = {
-            imageSource: undefined,
-        };
```

Comment:
This would render no image after the first render and only after add the `imageSource`
We don't have to use state for `imageSource` - it doesn't save us from re-renders, because most usages pass source like `<Image soruce={{ uri: '...' }} />` (a new object created inline)

Reply:
The `debounce` here was removed because it no longer has any effect (and was a workaround) as discussed here: https://github.com/Expensify/App/pull/9279#discussion_r1140746497

(Also it's inconsistent with the `.native` component where no debounce is used)

There are further changes that we can add to the Image component and I can open a follow up PR:
1. remove `configureOnLoad` - this was a workaround needed because at the time `react-native-web` did not provide dimension in the original `onLoad` callback
2. Turn `Image` into a functional component - no lifecycle and state are used. 

I do both of this changes here:
- https://github.com/Expensify/App/pull/13036

but the PR is on hold